packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -179,17 +179,9 @@ class RenderAndroidView extends PlatformViewRenderBox {
     Size targetSize;
     do {
       targetSize = size;
-      if (_viewController.isCreated) {
```

Comment:
This conditional logic moved back into `setSize`, so no longer needed to exist here.

Reply:
consider adding an assert, e.g. `assert(_viewController.isCreated); // created in setSize`

Reply:
I'm pretty sure that's not a valid assertion, for the same reason that you had added the `_isDisposed` check just after `create` in the code I'm removing: `setSize` is guaranteed to start creating it if necessary, but there's no guarantee that it's not then disposed later in the `await`.